### ISSUE TITLE
[BUG] iOS app not showing images deleted locally and sync icon missing from uploaded images

### ISSUE THREAD

**Issue by FrankelJb** [immich-app/immich#2069]
### The bug

I recently deleted thousands of local assets that have been successfully uploaded to Immich. After upgrading to 1.51.x , the assets are no longer visible in the iOS app. I've checked the web and all assets are there. I'm not sure what/where to check for the reason, I'll be happy to upload logs/configs etc.

Additionally, the "Backup" section has a negative remainder (currently reading -249). I can still upload assets successfully. However, the uploaded assets have the cloud with a line through it instead of showing the cloud with a tick icon. 

### The OS that Immich Server is running on

Kubernetes (RKE v1.26.1+rke2r1)

### Version of Immich Server

v1.51.2

### Version of Immich Mobile App

1.51.0 build.89

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Link to repo https://github.com/FrankelJb/home-ops/tree/main/kubernetes/apps/cloud/immich
```

### Your .env content

```Shell
DISABLE_REVERSE_GEOCODING: "true"
  LOG_LEVEL: "verbose"
  NODE_ENV: "development"
  REDIS_HOSTNAME: redis-master.default.svc.cluster.local
  IMMICH_SERVER_URL: http://immich-server.cloud.svc.cluster.local:3001
  IMMICH_WEB_URL: http://immich-web.cloud.svc.cluster.local:3000
  IMMICH_MACHINE_LEARNING_URL: http://immich-machine-learning.cloud.svc.cluster.local:3003
  TYPESENSE_HOST: immich-typesense.cloud.svc.cluster.local
  TYPESENSE_PORT: "8108"
  TYPESENSE_PROTOCOL: http
  TYPESENSE_DATA_DIR: /config
```

### Reproduction steps

```bash
No idea. I'm scared to delete the iOS profile at this point.
```

**Comment by fyfrey** [comment]
@FrankelJb could you please confirm that after logout and again loggin in in the app, the locally deleted (but uploaded) assets are now shown (having only the pure cloud icon)

**Comment by alextran1502** [comment]
> @FrankelJb could you please confirm that after logout and again loggin in in the app, the locally deleted (but uploaded) assets are now shown (having only the pure cloud icon)

@fyfrey i can confirm this

**Comment by FrankelJb** [comment]
Thanks for the fast feedback.

Remote assets are still missing. All assets show the crossed cloud.

**Comment by alextran1502** [comment]
Fixed in #2085
